docker/README.md

```diff
@@ -35,16 +35,16 @@ FF_GPU_BACKEND=<YOUR_GPU_BACKEND> ./docker/build.sh flexflow-environment
 ``` 
 
 ## Running a Docker container
-After having either built or downloaded a Docker container by following the instructions above, you can run it with the following command (the `flexflow` argument of the run script can be omitted):
+After having either built or downloaded a Docker container by following the instructions above, you can run it with the following command (the `--image_name flexflow` argument of the run script can be omitted). Moreover, you can choose the CUDA version to build using optional flag `--cuda_version`; otherwise, it will autodetect the cuda_version installed on the current machine and run the Docker container with it if available.
```

Comment:
can we update the readme to reflect the new way of passing the CUDA version, and image name?